src/solvers/refinement/string_constraint_generator_indexof.cpp

```diff
@@ -225,10 +241,23 @@ exprt string_constraint_generatort::add_axioms_for_last_index_of_string(
   return offset;
 }
 
-/// add axioms corresponding to the String.indexOf:(C), String.indexOf:(CI),
-/// String.indexOf:(String), and String.indexOf:(String,I) java functions
-/// \par parameters: function application with 2 or 3 arguments
-/// \return a integer expression
+/// Index of the first occurence of a target inside the string
+///
+/// If the target is a character:
+/// \copybrief add_axioms_for_index_of(const array_string_exprt&,const exprt&,const exprt&)
```

Comment:
Line too long. Please fix http://cbmc-linter.diffblue.com.s3-website-us-east-1.amazonaws.com/linter-b07fcddda78d14adffe806abd4891189902d7eb0.txt

Reply:
I'm not sure how to fix this, breaking the line breaks the doxygen link, and adding a `// NOLINT` comment will make `NOLINT` appear in the doxygen documentation.

Reply:
Does the `NOLINTNEXTLINE` show up in the doxygen? I'd strongly prefer a error in the linter to confusion in the documentation...

Reply:
No because there are only two `/`s (not three) at the beginning of the line. From what I've seen, the output is fine.